core/iwasm/include/wasm_export.h

```diff
@@ -275,6 +275,13 @@ typedef struct wasm_val_t {
 } wasm_val_t;
 #endif
 
+/* Global instance*/
+typedef struct wasm_global_inst_t {
+    wasm_valkind_t kind;
+    bool is_mutable;
```

Comment:
personal opinion:

``` c
typedef struct wasm_global_inst_t {
  wasm_global_type_t type;
  void *data;
}
```

Reply:
@lum1n0us since wasm_global_type_t is a pointer, I think this would require a memory allocation and the corresponding ownership issues. Also, the underlying non-pointer WASMGlobalType is not currently exposed, it's internal only. Is it worth making the necessary changes to avoid duplicating those two members?

Reply:
Yes, wasm_global_type_t is an internal type, unless we add the declarations in wasm_export.h, but it seems no need to do that.